### PR TITLE
whitelist theano

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist theano

### Linked issues
Progresses #1901 

### Functionality 
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -22808,6 +22808,363 @@
...
+    "theano.tensor.tests.test_type_other": [],
+    "theano.tensor.tests.test_utils": [],
+    "theano.tensor.tests.test_var": [],
+    "theano.tensor.tests.test_xlogx": [],
+    "theano.tensor.type": [],
+    "theano.tensor.type_other": [],
+    "theano.tensor.utils": [],
+    "theano.tensor.var": [],
+    "theano.tensor.xlogx": [],
+    "theano.tests": [],
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:23136]: remove test packages, it's noise

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:23136]: done

Comment (nfx) [reply src/databricks/labs/ucx/source_code/known.json:23136]: image
they're in multiple places. please review diffs before opening PRs :)

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: Done

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**